src/components/JobConfiguration.vue

```diff
@@ -328,7 +416,19 @@ ion-list {
 
   .mobile-only {
     display: none;
-  }  
+  }
+  .job-actions {
+    margin-top: var(--spacer-sm);
```

Comment:
Please arrange the usage of css properties in correct way. Display should be written first and margin will be on last.

Reply:
Arranged ma'am